amazon/ion/writer_binary_raw.py

```diff
@@ -226,6 +233,16 @@ def _serialize_lob_value(event, tid):
 
 _MICROSECOND_DECIMAL_EXPONENT = -6  # There are 1e6 microseconds per second.
 
+_TEN_EXP_MINUS_ONE = [
+    -1,
+    9,
+    99,
+    999,
+    9999,
+    99999,
+    999999,
+]
+
```

Comment:
What is this constant for? Probably warrants a comment and should probably be a `tuple` instead of `list`.

Reply:
This constant was here originally, before I started making changes. It checks if `fractional_precision` has too few digits to support the current `microsecond` precision [here.](https://github.com/amzn/ion-python/pull/100/files/c44ce224c1e66cec7c3dd9fc7f8746e447fb73cf..788adafde6ad54312adc3330e2d1abc1edbfe723#diff-d98fe681a3236eae60289fa56a2a0083R286) I removed it before, but we still need it now in case we have a normal `datetime` with no `fractional_seconds`to write.

Reply:
The fact that it's not a `tuple` is my fault as the original author. I agree that a `tuple` is more suitable.

Reply:
Done, changed to tuple.